module/apmelasticsearch/client.go

```diff
@@ -63,17 +63,23 @@ type roundTripper struct {
 func (r *roundTripper) RoundTrip(req *http.Request) (*http.Response, error) {
 	ctx := req.Context()
 	tx := apm.TransactionFromContext(ctx)
+	traceContext := tx.TraceContext()
 	if tx == nil || !tx.Sampled() {
```

Comment:
I think we need to split the `if tx == nil` case out, and not call `apmhttp.SetHeaders` in that case?